packages/remix/test/integration/package.json

```diff
@@ -19,6 +19,7 @@
     "@remix-run/dev": "^1.6.5",
     "@types/react": "^17.0.47",
     "@types/react-dom": "^17.0.17",
+    "nock": "^13.1.0",
```

Comment:
did we have to add this dep?

Reply:
TS falls back to the global installation of `nock`, which seems to have a slight difference between what we use in `node-integration-tests`. Had to add this (the same version with `node-integration-tests`) to resolve it.